flask/cli.py

```diff
@@ -41,10 +41,17 @@ def find_best_app(module):
 
     if len(matches) == 1:
         return matches[0]
+
+    # Finally, look for a factory function
+    factory = getattr(module, 'create_app', None)
+    if callable(factory):
```

Comment:
I think this check is not very pythonic. Especially considering how `create_app` is a name only appropriate for something callable in the first place.